app/src/main/res/xml/preferences.xml

```diff
@@ -95,6 +95,10 @@
         android:title="@string/pref_zoom_dialog"/>
   </PreferenceCategory>
 
+  <Preference
+      android:key="pref_text_size"
```

Comment:
I think you must first `rebase` your branch because the indentation of all the `xml` files have been improved, which does not reflect in your PR.